website/docs/feature-flag-tutorials/python/implementing-feature-flags.md

```diff
@@ -0,0 +1,331 @@
+
+
+Create 1 or more new surveys so they’re populated in your database!
+
+## 5. Use a feature flag to release a delete method
+
+In a real-world use case for your feature flag, you can roll out new features to users by configuring the flag's strategy.
+
+In this case, our app currently supports creating a survey, but once we create one, we can’t get rid of it. We want to roll out a ‘delete’ button in our list of surveys to all users so we have the option to remove them from our database.
+
+This will require us to:
+- Create a new route in our app
+- Create a method that deletes a survey based on survey ID
+- Create a delete button
+- Map the delete button to the delete method
+
+In `src/backend/surveys/routes.py`, add `client` to the existing `backend` import statement. The full import line will now look like this:
+
+
+```py
+from backend import db, client
+```
+
+We’ve imported the initialized Unleash client into `routes.py`. Now we can use that data to pass into the `surveys_list_page` method. This will allow us to check the status of the enabled flag to conditionally render the 'Delete' button on the surveys page.
+
+Add `client` as a parameter in the template that we return in the `surveys_list_page` method.
+
+The modified return statement in this method will now look like this:
+
+```py
+return render_template("surveys_list.html", surveys=surveys, client=client)
+```
+
+In the same file, we will create a new route and a ‘delete’ method with this code snippet:
+
+```py
+@bp.route("/surveys/<int:survey_id>/delete", methods=["GET", "POST", "DELETE"])
+def delete_survey(survey_id):
+   survey = db.get_or_404(Survey, survey_id)
+   db.session.delete(survey)
+   db.session.commit()
+
+   return redirect(url_for("surveys.surveys_list_page"))
+```
+
+The server now has a route that uses a survey ID to locate the survey in the database and delete it.
+
+To make calls to this route, we will create a delete button for each survey on the page.
+
+In `src/backend/templates/surveys_list.html`, add the following code to your survey table underneath the “Survey Page” button:
+
+```html
+{% if client.is_enabled('delete_survey_flag') %}
+<td class="text-end"><a href="{{ url_for('surveys.delete_survey', survey_id=survey.id) }}" class="btn btn-sm btn-danger">Delete</a></td>
+{% endif %}
+```
+
+This code wraps a delete button in a conditional statement that checks whether or not the feature flag is enabled. This button has a link that points to the `delete_survey` method we created, which will pull in the survey using an ID to search the database, find the matching survey, and delete it from the database session.
+
+Your surveys page will now look something like this:
+
+
+![Screenshot of app in browser with delete buttons in survey table](/img/python-tutorial-surveys-with-delete.png)
+
+
+Test the new functionality by deleting one of your surveys. The page should refresh without the survey you deleted.
```

Comment:
If I understood correctly, this only hides the button in the UI. Might be worth adding the `if client.is_enabled('delete_survey_flag')` check to the route as well, so that users are not able to delete things by directly hitting the endpoint, and I think it would prove to be more interesting for a tutorial like this, where the focus is on a backend SDK.